### PR TITLE
Adjusted Calculations for Would-Be Costs and True Costs

### PR DESCRIPTION
**Pull request by aivantg**
[//]: # "These comments meant for your reference, they are invisible and don't need to be deleted"

## What's new in this PR?

Added Calculations for True Costs + Would-Be Costs based on Grayson's Email
Cleaned up Utility API file

[//]: # "Describe what's new in this PR"

### How to Review

[//]: # 'The order in which to review files and what to expect when testing locally'

### Related PRs

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc"

### Next steps

[//]: # "What doesn't work yet, what's NOT in this PR/has to be done "
tbh this should hopefully just be it, need to get grayson's approval

### Tests Performed, Edge Cases

[//]: # 'If you made changes/additions to `airtable.js` make sure you update/add'
[//]: # '[at least 1] corresponding unit tests in `airtable.spec.js`'

### Screenshots

[//]: # "Add screenshots!!! If you'd like"

CC: @dfangshuo

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
     "react-dom": "^16.13.1"
   },
   "scripts": {
-    "generate-schema": "generate-airtable-schema && pretty-quick",
+    "add-require-to-airtable": "(echo \"require('dotenv-safe').config();\" && cat airtable/airtable.js) > temp.js && mv temp.js airtable/airtable.js",
```
Comment (dfangshuo) [inline package.json:25]: hmm is there a reason why we need to do this and can't just write these lines into the airtable.js file itself?

File: utils/chartGeneration.js [hunk 2]
```diff
@@ -41,11 +41,15 @@ const generateCostOverTimeChart = (prevBills, latestBill) => {
...
   const monthData = bills.map(b => b.startDate);
-  // TODO: account for pge charges + ebce charges
   const trueCosts = bills.map(
-    b => b.currentCharges - b.ebceRebate - b.estimatedRebate
+    b =>
+      b.ebceCharges +
+      b.pgeCharges +
+      b.currentCharges -
+      b.ebceRebate -
+      b.estimatedRebate
```
Comment (dfangshuo) [inline utils/chartGeneration.js:50]: clean ++ :')

File: utils/pdf/BillPageOne.js [hunk 3]
```diff
@@ -19,7 +19,8 @@ export default class BillPageOne extends React.PureComponent {
             amountReceived: 0,
             balance: 0
           };
-    const round = x => parseFloat(x).toFixed(3);
+    const round = (x, y = 2) => parseFloat(x).toFixed(y);
+    const roundAndCut = (x, y = 2) => Math.floor(x * 10 ** y) / 10 ** y;
     const formatAirtableDate = d =>
```
Comment (dfangshuo) [inline utils/pdf/BillPageOne.js:24]: might be helpful to move these into a separate mathUtils file or smth?

File: package.json (hunk 1 above)
Comment (aivantg) [reply package.json:25]: because these lines only need to be added to the node version of this. If we added to airtable.js, they get overwritten by the schema generator everytime it is regenerated
